src/client/app/gear-availability-report/gear-availability.controller.js

```diff
@@ -38,12 +38,19 @@
     vm.openStartPicker = openStartPicker;
     vm.openEndPicker = openEndPicker;
     vm.validDates = validDates;
+    vm.authorized = false;
     activate();
     
     /////////////////////
 
     function activate() {
-      logger.info("Activated Create Request");
+      var profile = ProfileAccess.getProfile();
+      if(profile) {
+        vm.authorized = profile.role_id == 3;

```

Comment:
I agree, that would probably be a better way to handle it. I was pretty much just following the example that US-11 (the trend reports) set.

I spent a few hours looking into it this morning and I'm kind of stuck on how to handle it. I found this FAQ for the ui-router that sounded like what we need but I'm not sure if it's applicable or how it would fit into our project:
https://github.com/angular-ui/ui-router/wiki/Frequently-Asked-Questions#how-to-create-rules-to-prevent-access-to-a-state

Do you have any ideas for how we could make this work?